ACE/ace/CDR_Base.h

```diff
@@ -207,6 +207,14 @@ class ACE_Export ACE_CDR
   typedef ACE_UINT32 ULong;
   typedef ACE_INT64 LongLong;
   typedef ACE_UINT64 ULongLong;
+  typedef ACE_INT8 Int8;
+  typedef ACE_UINT8 UInt8;
+  typedef Short Int16;
+  typedef UShort Uint16;
+  typedef Long Int32;
+  typedef ULong Uint32;
```

Comment:
`UInt32` and `UInt64` to keep it all consistent

Reply:
Thought I did that already, but I guess it was just the int8 ones.